### ISSUE TITLE
test failure on latest

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa-content-server#2796]
Following on from #2795. I'll run it a couple of times to see if this repeats.

```
[21:15:08] [38;5;10m✓ firefox on any platform - oauth sign up verification_redirect - signup, same browser different window, verification_redirect=always [37m (9.024s) [0m
[21:15:18] [38;5;10m✓ firefox on any platform - oauth sign up verification_redirect - signup, verify different browser, verification_redirect=always [37m (8.039s) [0m
[21:15:57] [38;5;9m× firefox on any platform - oauth reset password - reset password, verify same browser [37m (36.967s)
[21:15:57] [31mNoSuchElement: [POST http://localhost:4444/wd/hub/session/426754d0-64cf-49b0-acc9-fe4562cc5a2e/element / {"using":"css selector","value":"#loggedin"}] Unable to locate element: {"method":"css selector","selector":"#loggedin"}
[21:15:57]For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
[21:15:57]Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:16:47'
[21:15:57]System info: host: 'ip-172-31-11-113', ip: '172.31.11.113', os.name: 'Linux', os.arch: 'amd64', os.version: '3.13.0-51-generic', java.version: '1.7.0_80'
[21:15:57]Driver info: driver.version: unknown
[21:15:57]  at ProxiedSession._post  <node_modules/intern/node_modules/leadfoot/Session.js:59:30>
[21:15:57]  at ProxiedSession.Session.find  <node_modules/intern/node_modules/leadfoot/Session.js:1055:15>
[21:15:57]  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:42:36>
[21:15:57]  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
[21:15:57]  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
[21:15:57]  at processImmediate [as _immediateCallback]  <timers.js:354:15>
[21:15:57]  at Command.find  <node_modules/intern/node_modules/leadfoot/Command.js:23:10>
[21:15:57]  at Command.prototype.(anonymous function) [as findByCssSelector]  <node_modules/intern/node_modules/leadfoot/lib/strategies.js:24:16>
[21:15:57]  at Test.registerSuite.reset password, verify same browser [as test]  <tests/functional/oauth_reset_password.js:118:10>
[21:15:57]  at start.then.finally.self.timeElapsed  <node_modules/intern/lib/Test.js:211:24>
[21:15:57]  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
[21:15:57]  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
[21:15:57]  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
[21:15:57]  at Object.run [as _onImmediate]  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
[21:15:57]  at processImmediate [as _immediateCallback]  <timers.js:354:15> [0m
[21:16:06] [38;5;10m✓ firefox on any platform - oauth reset password - reset password, verify same browser with original tab closed [37m (8.099s) [0m
```

**Comment by jrgm** [comment]
Okay, failed again. => train-42

```
[21:46:35][Step 1/1]  [38;5;10m✓ firefox on any platform - oauth sign up verification_redirect - signup, same browser, original tab closed, verification_redirect=always [37m (8.131s) [0m
[21:46:44][Step 1/1]  [38;5;10m✓ firefox on any platform - oauth sign up verification_redirect - signup, same browser different window, verification_redirect=always [37m (7.51s) [0m
[21:46:52][Step 1/1]  [38;5;10m✓ firefox on any platform - oauth sign up verification_redirect - signup, verify different browser, verification_redirect=always [37m (6.772s) [0m
[21:47:31][Step 1/1]  [38;5;9m× firefox on any platform - oauth reset password - reset password, verify same browser [37m (36.39s)
[21:47:31][Step 1/1]  [31mNoSuchElement: [POST http://localhost:4444/wd/hub/session/76c8b1db-2a2c-4379-84d2-31973d40e5ab/element / {"using":"css selector","value":"#loggedin"}] Unable to locate element: {"method":"css selector","selector":"#loggedin"}
[21:47:31][Step 1/1] For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
[21:47:31][Step 1/1] Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:16:47'
[21:47:31][Step 1/1] System info: host: 'ip-172-31-11-113', ip: '172.31.11.113', os.name: 'Linux', os.arch: 'amd64', os.version: '3.13.0-51-generic', java.version: '1.7.0_80'
[21:47:31][Step 1/1] Driver info: driver.version: unknown
[21:47:31][Step 1/1]  at ProxiedSession._post <node_modules/intern/node_modules/leadfoot/Session.js:59:30>
[21:47:31][Step 1/1]  at ProxiedSession.Session.find <node_modules/intern/node_modules/leadfoot/Session.js:1055:15>
[21:47:31][Step 1/1]  at Command.<anonymous> <node_modules/intern/node_modules/leadfoot/Command.js:42:36>
[21:47:31][Step 1/1]  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
[21:47:31][Step 1/1]  at Object.run [as _onImmediate] <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
[21:47:31][Step 1/1]  at processImmediate [as _immediateCallback] <timers.js:354:15>
[21:47:31][Step 1/1]  at Command.find <node_modules/intern/node_modules/leadfoot/Command.js:23:10>
[21:47:31][Step 1/1]  at Command.prototype.(anonymous function) [as findByCssSelector] <node_modules/intern/node_modules/leadfoot/lib/strategies.js:24:16>
[21:47:31][Step 1/1]  at Test.registerSuite.reset password, verify same browser [as test] <tests/functional/oauth_reset_password.js:118:10>
[21:47:31][Step 1/1]  at start.then.finally.self.timeElapsed <node_modules/intern/lib/Test.js:211:24>
[21:47:31][Step 1/1]  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
[21:47:31][Step 1/1]  at runCallbacks <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
[21:47:31][Step 1/1]  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
[21:47:31][Step 1/1]  at Object.run [as _onImmediate] <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
[21:47:31][Step 1/1]  at processImmediate [as _immediateCallback] <timers.js:354:15> [0m
[21:47:39][Step 1/1]  [38;5;10m✓ firefox on any platform - oauth reset password - reset password, verify same browser with original tab closed [37m (7.326s) [0m
```

**Comment by vladikoff** [comment]
It passed now, we have some race condition, from my investigation it is either something to do with the window manager (windows are too slow to close) or this: https://github.com/mozilla/fxa-content-server/issues/2483
